From 825a77fc23af418dd0a98101dc94ec930fad3456 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Fri, 15 Aug 2025 23:20:01 +0800
Subject: [PATCH 37/63] loonggpu: Adapt for the rename of
 DRM_GPU_SCHED_STAT_NOMINAL to DRM_GPU_SCHED_STAT_RESET

Link: https://git.kernel.org/torvalds/c/0a5dc1b67ef5
Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 include/loonggpu_helper.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/loonggpu_helper.h b/include/loonggpu_helper.h
index 8f7715c..6340701 100644
--- a/include/loonggpu_helper.h
+++ b/include/loonggpu_helper.h
@@ -432,7 +432,11 @@ static inline int lg_atomic_read_drm_open_count(struct drm_device *dev)
 
 #if defined(LG_DRM_SCHED_BACKEND_OPS_TIMEDOUT_JOB_RET_SCHED_STAT)
 #define lg_loonggpu_job_timedout_ret enum drm_gpu_sched_stat
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 17, 0)
+#define LG_DRM_GPU_SCHED_STAT_NOMINAL DRM_GPU_SCHED_STAT_RESET
+#else
 #define LG_DRM_GPU_SCHED_STAT_NOMINAL DRM_GPU_SCHED_STAT_NOMINAL
+#endif
 #else
 #define lg_loonggpu_job_timedout_ret void
 #define LG_DRM_GPU_SCHED_STAT_NOMINAL
-- 
2.52.0

